pkg/composableschemadsl/parser/parser.go

```diff
@@ -584,3 +588,50 @@ func (p *sourceParser) tryConsumeNilExpression() (AstNode, bool) {
 	defer p.mustFinishNode()
 	return node, true
 }
+
+func (p *sourceParser) consumeImport() AstNode {
+	importNode := p.startNode(dslshape.NodeTypeImport)
+	defer p.mustFinishNode()
+
+	// from ...
+	p.consumeKeyword("from")
+
+	// Consume alternating periods and identifiers
+	for {
+		if _, ok := p.consume(lexer.TokenTypePeriod); !ok {
+			return importNode
+		}
+
+		segmentNode, ok := p.tryConsumeIdentifierLiteral()
```

Comment:
This isn't a try consume; it should be a consume, because a missing identifier here should be an error. Please add a test case for this

Reply:
I was going off of the implementation of this function, which does seem to use `consume`: https://github.com/authzed/spicedb/blob/c8e739ce370133d6e99ea20651f98f9f39160eb7/pkg/composableschemadsl/parser/parser.go#L576

I'll add a test case, though.